provisioning-daemon/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=provisioning-daemon
+PKG_VERSION:=1.0.0
+#PKG_VERSION:=dev
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/provisioning-daemon
```

Comment:
The source URL should be remote i.e. https://github.com/CreatorDev/ci40-provisioning-daemon.git

Developers should set the source link locally for development purpose